### PR TITLE
Feature/di

### PR DESCRIPTION
**Pull request by TekaLishoo**
Pattern dependency injection + complete parsing

### CODE REVIEW

File: routers/twitch_router.py [hunk 1]
```diff
@@ -13,7 +14,8 @@ def list_of_objs():
     """
     Return all twitch objects in a database
     """
-    streams = database.streams
+    client = Mongo(os.environ['MONGO_USER'], os.environ['MONGO_PASS'], os.environ['MONGO_DB'])
+    streams = client().streams
```
Comment (achopik) [inline routers/twitch_router.py]: You can encapsulate os.environ[...] inside DAO class. So you would use values by default inside

File: routers/lamoda_router.py [hunk 2]
```diff
@@ -14,17 +14,9 @@ def get_products():
...
-        name='pillow',
-        price=230.1,
-        descroption={'color': 'red', 'material': 'cotton'}
-    )
-    products = database.products
-    products.insert_one(product.dict())
-    return paginate(products)
+    database = ContainerMongo().mongo
+    products = database.client.products
+    return ContainerMongo().get_list_lamoda_products()
```
Comment (achopik) [inline routers/lamoda_router.py]: Use Depends() for injecting dependencies inside a coroutine

File: parsing_scripts/twitch_parser.py [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
     BASE_URL = "https://api.twitch.tv/helix/streams"
 
-    async def parse(self):
+    async def parse(self, k: Kafka, mongo: ContainerMongo):
         soup = await self.get_soup(self.BASE_URL)
+        for i in range(10):
+            d = k.producer.send('topic_twitch', value={'counter': 'twitch'})
+            data = d.get()
+            print(f'send {data}')
+            await asyncio.sleep(1)
```
Comment (achopik) [inline parsing_scripts/twitch_parser.py]: you can provide Kafka class inside __init__, or configure() method for instance and remove it from parse method signature, so you would rather do self.kafka.producer.send(...)

**Review by achopik — CHANGES_REQUESTED** [review]

File: routers/twitch_router.py (hunk 1 above)
Comment (TekaLishoo) [reply routers/twitch_router.py]: Done according to lamoda router schema

File: routers/lamoda_router.py (hunk 2 above)
Comment (TekaLishoo) [reply routers/lamoda_router.py]: Done

File: parsing_scripts/twitch_parser.py (hunk 3 above)
Comment (TekaLishoo) [reply parsing_scripts/twitch_parser.py]: Done with init def

**Pull request merged**